.github/workflows/auto-assign.yml

```
@@ -0,0 +1,32 @@
+name: Auto-assign maintainer
+
+on:
+  issue_comment:
+    types:
+      - created
+  pull_request_review_comment:
+    types:
+      - created
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  assign:
+    # Only run on PR comments
+    if: github.event.issue.pull_request
+    runs-on: ubuntu-slim
+    timeout-minutes: 5
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          sparse-checkout: |
+            .github
+      - uses: actions/github-script@ed597411d8f924073f98dfc5c65a23a2325f34cd # v8.0.0
+        with:
+          script: |
+            const script = require('./.github/workflows/auto-assign.js');
+            await script({ github, context });
```

```suggestion
      - uses: actions/github-script@ed597411d8f924073f98dfc5c65a23a2325f34cd # v8.0.0
        with:
          script: |
            ...
```

The amount of js code is not large. Let's put it directly in github-script to skip checkout.